### ISSUE TITLE
No Data

### ISSUE THREAD

**Issue by ypriverol** [ypriverol/containers-rules-manuscript#10]
No data should be included in the container.

**Comment by prvst** [comment]
I would rephrase to something like "Don't use an image to maintain data". Since we have to push the data inside the containers to be processed, the way you stated might confuse people who never managed them before.

**Comment by ypriverol** [comment]
@prvst Can you review the current version of the manuscript and include this idea. ?

**Comment by pditommaso** [comment]
I agree on this. The comment suggest it has been addressed however I don't see it as an explicit point.

**Comment by ypriverol** [comment]
@pditommaso I think this has been miss in some way in one of the versions. I will put this inside the ```size of containers``` as a subsection.

**Comment by pditommaso** [comment]
That's a good option. I would also stress also the fact that container are meant to make software portable and easy deployable across platforms. Data is portable by definition, on the other hand putting a piece of data inside a container limits the accessibility to a container only environment. 

(I would even take in consideration to have it as a separate point)

**Comment by ypriverol** [comment]
I have updated the current version of the paper including the data in a separate point (https://github.com/ypriverol/containers-rules-manuscript/blob/master/manuscript.adoc#6-dont-use-a-container-to-maintain-data-no-data-inside-the-container). Can you reviewed and make a PR with your thoughts. @pditommaso

**Comment by susheel** [comment]
👍

**Comment by ypriverol** [comment]
This has been added as a Recommendation now and can be close .. if anyone has more things to add, please do a PR and we continue the discussion there.
